src/analysis/processing/qgsalgorithmjoinbylocation.cpp

```diff
@@ -0,0 +1,368 @@
+/***************************************************************************
+                         qgsalgorithmjoinbylocation.cpp
+                         ---------------------
+    begin                : November 2019
+    copyright            : (C) 2019 by
```

Comment:
You can fill this in with your details and take credit for your work :smile:

Reply:
I know, I'm still on the fence about what to write.